helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -408,3 +408,54 @@ configMaps: []
   #     key: |
   #       this is the content of the configmap
   #       when mounted, key will be the name of the file
+
+# specify the fe proxy deploy or not.
+feProxy:
+  # specify the fe proxy deploy or not.
+  enabled: true
+  replicas: 1
+  image:
```

Comment:
can we make this image section omit by default?

Reply:
ok, I will check these fields, and make them have default values.